waterloop-site/src/components/SustainableTech/Hero/Hero.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import LandingImage from '../../../static/img/sustainable-tech/HeroLandingImage.svg';
+import GreenTechLogo from '../../../static/img/sustainable-tech/GreenTechLogo.svg';
+
+const LandingContainer = styled.div`
+  background-image: url(${LandingImage});
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  background-color: #e5f6fa;
+  width: 100vw;
+  height: 100vh;
+`;
+
+const MainContent = styled.div`
+  position: absolute;
+  top: 38%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`;
+
+const GreenTech = styled.img`
+  height: 300px;
+  width: auto;
+
+  @media (max-width: 1200px) {
+    height: 250px;
+  }
+
+  @media (max-width: 430px) {
+    height: 190px;
+  }
+
+  @media (max-width: 300px) {
+    height: 150px;
+  }
+`;
+
+const ButtonContainer = styled.button`
+  border-radius: 30px;
+  background-color: #203d7a;
+  text-decoration: none;
+  color: white;
+  font-family: 'IBM Plex Sans';
+  font-size: 19px;
+  font-weight: 500;
+  padding: 15px 15%;
+  border: #203d7a;
+  margin: 10% 0;
+  font-weight: 600;
+
+  &:hover {
+    cursor: pointer;
+    color: #203d7a;
+    background-color: white;
+  }
+
+  @media (max-width: 650px) {
+    font-size: 17px;
+    padding: 10px 15%;
+  }
+`;
+
+const Hero: React.FC = () => (
+  <LandingContainer id="outer">
+    <MainContent id="home-scroll">
+      <GreenTech alt="Green Tech Logo" src={GreenTechLogo} />
+      <ButtonContainer
+        as="a"
+        href="https://tinyurl.com/uwgreentech"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        SIGN UP
```

Comment:
We have 2 sign-up buttons (one in the navbar + hero) which is consistent with the Figma design. However, on my laptop screen, the GreenTech logo is slightly cut off. Maybe we could delete the button in the hero and move down the logo? 

<img width="1440" alt="Screen Shot 2022-03-08 at 10 30 02 AM" src="https://user-images.githubusercontent.com/68434174/157270044-b54e1120-b90d-4cea-a0c0-08aacb73da73.png">

Reply:
I agree, I've made a corresponding change that I'll push later.

Reply:
I think this is happening because the size of the logo changes with width, not with height, so on screens with a smaller height the logo is a bit too large

Reply:
From my end I think this can be fixed by adding 
`@media (max-height: 670px){
    height: 200px;
  }

  @media (max-height: 580px){
    height: 170px;
  }`

to `GreenTech = styled.img` at `/SustainableTech/Hero/Hero.tsx`

Reply:
Just pushed the change you suggested, thanks!